rocketpy/mathutils/function.py

```diff
@@ -2176,7 +2182,7 @@ def __radd__(self, other):
         """
         return self + other
 
-    def __sub__(self, other):
+    def __sub__(self, other):  # pylint: disable=too-many-statements
         """Subtracts from a Function object and returns a new Function object
```

Comment:
Do we need to have the __sub__ defined like this? It used to just call __add__ and __mul__

I guess you are avoiding performing some checks more than once?

Reply:
I preferred to define the `sub`, since the "correct" definition would call add and neg (or mul, as you mentioned). This would make only the `sub` operation take about twice the time. Even though that may be a meaningless time save, I believe, for consistency, it makes sense to have all of them take about the same time.